### PR TITLE
Fix #298; Added global time measurement

### LINKED ISSUE #298: Measure training time across multiple epochs
So far training time is just measured on epoch level. To be able to measure the effect of some change globally across multiple epochs, we need to stopwatch it.

### PR DESCRIPTION
**Pull request by tilmankamp**

### CODE REVIEW

File: DeepSpeech.ipynb [hunk 1]
```diff
@@ -1680,6 +1680,11 @@
    "outputs": [],
    "source": [
     "def train():\n",
+    "    \n",
```
Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Why this extra return?

File: DeepSpeech.ipynb [hunk 2]
```diff
@@ -1720,8 +1725,10 @@
     "                             epoch == epochs - 1\n",
     "\n",
     "        print \"Training model...\"\n",
+    "        global_train_time = stopwatch(global_train_time)\n",
     "        train_time = stopwatch(train_time)\n",
     "        result = calculate_loss_and_report(train_context, session, epoch=epoch, query_report=is_display_step)\n",
+    "        global_train_time = stopwatch(global_train_time)\n",
```
Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Just a strange question, why the ordering
```python
  global_train_time = stopwatch(global_train_time)
  train_time = stopwatch(train_time)
  result = calculate_loss_and_report(train_context, session, epoch=epoch, query_report=is_display_step)
  global_train_time = stopwatch(global_train_time)
  train_time = stopwatch(train_time)
```
and not

```python
  global_train_time = stopwatch(global_train_time)
  train_time = stopwatch(train_time)
  result = calculate_loss_and_report(train_context, session, epoch=epoch, query_report=is_display_step)
  train_time = stopwatch(train_time)
  global_train_time = stopwatch(global_train_time)
```

**Comment by tilmankamp** [comment]
Followed up on all your comments.

**Comment by kdavis-mozilla** [comment]
LGTM

**Pull request merged**